### ISSUE TITLE
Photos uploaded are consuming quota

### ISSUE THREAD

**Issue by rareseu** [Kingsman44/Pixelify#166]
I installed it, and Photos app can definitely see that I am using Pixel XL because it says I have unlimited backups in Original Quality & in backup settings I cant change that (for Storage Quality).
But after backing up some photos, in web (and after that in app too) I can see that it was uploaded from Android device, but is taking space...

**Comment by Kingsman44** [comment]
Your rom?, android version?

**Comment by FR46M3N7-P4R71CL3** [comment]
Yup, this seems to be a recent Google change. If you aren't affected, in just a few days, everyone will. This is happening across ROMs, android versions and surviving reinstalls of pixelify. Even older Google photos client doesn't seem to work, It is showing unlimited in app settings, however, uploaded media is counted against storage.

Reinstalling temporarily works (may be for an hour or so) before they start to count.
cc: @Kingsman44

**Comment by AbhayF8** [comment]
same here

**Comment by Sappstal** [comment]
I've been using version 2.0.2 for almost a month and photos/videos never consumed quota. I've just updated to v2.1 to try, but seems working too.
I'm on Miui.eu 13.0.5.0 (I'll update the comment after a while to test if it still works)

**Comment by I-Iawk** [comment]
> Your rom?, android version?

Just noticed this now... Stock Android 11.

**Comment by PanosChtz** [comment]
Confirmed that photos/videos now count against quota as well.
Tried fresh installing pixelify 2.1 and 2.2 on NoxPlayer and Bluestacks emulators but nothing worked.

**Comment by PanosChtz** [comment]
Some possible things to check:

1. Change spoofing of photos into Pixel 5 and backup as storage saver?
2. Check if device certification in google play plays a role? (since I was using emulator, my device was not certified).
